packages/bot-web-ui/src/components/load-modal/load-modal.tsx

```diff
@@ -26,7 +26,7 @@ const LoadModal = observer(() => {
     const { setPreviewOnPopup } = dashboard;
     const header_text = localize('Load strategy');
 
-    if (isMobile()) {
+    if (ui.is_mobile) {
```

Comment:
since we are writing `const { is_mobile } = ui` across the whole codebase, lets do that here aswell and write just is_mobile inside the if condition instead of ui.is_mobile @sandeep-deriv

Reply:
done @farabi-deriv ✅